### DISCUSSION TITLE
Bootloader for atmega2560 using UNO

### DISCUSSION THREAD

**Discussion by Manjusha1510** [nodeschool/discussions#2616] [Q&A]
I have developed a board based on atmega2560. Two boards are working as desired but now in third board, during uploading a bootloader; it comes out with an error. Now if I try to do any thing I get an error Device signature = 0x000000. If I do not load bootloader, the sketch gets uploaded and works fine with default clock i.e. 1Mhz. I am using UNO board as programmer. 
 I had changed the boards.txt file mega.menu.cpu.atmega2560.bootloader.high_fuses=0xD9.
please help me to solve this issue.

**Comment by tonhowtf** [comment]
Yes, but, why?

**Comment by DURGESH716** [comment]
Because a board running the Micronucleus bootloader can be reprogrammed without the use of the RESET pin, it is practical to use them with reset configured to act as an I/O pin. This is, obviously, of particular utility on the ATtiny85, where available pins are at a premium, but may be used on any part (except the ATtiny841, in the current version of this core). Because this requires changing the fuses, it can only be enabled with an ISP programmer. Once reset as been disabled, it can no longer be programmed via ISP. Only HVSP or parallel programming (depending on the part - each part supports one or the other, never both), wherein 12v is applied to the reset pin under specific conditions, can be used to unset this fuse or to unbrick a part where reset is disabled and the bootloader is no longer functioning. Note that the bootloader can still be upgraded over USB - including to a version that only runs upon external reset!
